woocommerce/woocommerce-ios
# Widgets: Update Xcode configuration

# Why 

After merging the main Widgets target(#7558), we started to experience some issues when [building the project into a device](https://a8c.slack.com/archives/CC7L49W13/p1662368046975199) and when [submitting builds](https://a8c.slack.com/archives/C6H8C3G23/p1662366847619799) to apple. This PR attempts to fix those problems by:

- Manually handling the `StoreWidgets` development provisioning profile like we do for the `WooCommerce` target.
<img width="504" alt="manual-signin" src="https://user-images.githubusercontent.com/562080/188475478-aeffe2e8-c4b1-4500-894b-1f4c7646ef8d.png">

- Disabling `Bitcode` for the `StoreWidgets` target like we do for the `WooCommerce` target.
<img width="536" alt="bitcode" src="https://user-images.githubusercontent.com/562080/188475807-ba2f6a3d-37d2-49b1-bd0b-933033ed9a84.png">

- Removing the Widget default Custom Intent as we won't have plans to build any configurable widget.
<img width="439" alt="custom-intent" src="https://user-images.githubusercontent.com/562080/188476113-f8c354c3-b764-4606-950b-8499ff1db1b9.png">

# Testing Instructions

- See that you can build & run the app successfully on a device.
- Kill & re-run the app.
- See that the Store Info Widget can be added (Template data for now)


---
